.github/workflows/slack.yml

```diff
@@ -1,4 +1,4 @@
-name: Slack Notify
+name: Slack Notifier on Star
```

Comment:
The change in name here will cause inconsistency with the rest of the repos. The name change is fine, but it would be good to have this consistent across all repos - one way or the other.

Reply:
@leecalcote No problem, I noticed that the other adapters repo, had `Slack Notifier on Star`, which is why I suggested to change here.

See here on https://github.com/acald-creator/meshery-consul/blob/master/.github/workflows/slack.yml

While I typed `Notifier` wrong trying to type out the word, but in `meshery-istio` can be consistent with `Slack Notify on Star`

I can still update to make it `Slack Notify on Star`, didn't notice I typed the word out wrong.

Reply:
Oh, I see. Makes much sense. Yes, a quick change to “Notify” from “Noitifier” is probably worth the few moments it will take to update. 👍

Reply:
```suggestion
name: Slack Notify on Star
```